ci3/base/denoise

```diff
@@ -7,9 +7,9 @@ if [ "$#" -eq 0 ]; then
   exit 1
 fi
 
-if [[ -n "${BUILD_SYSTEM_DEBUG:-}" || "${DENOISE:-0}" -eq 0 ]] ; then
+if [ "${DENOISE:-0}" = 0 ]; then
   set -e
-  eval "$*"
```

Comment:
Why changing this back to `$@`?
If you want to do e.g. `denoise "true && true"` (i.e. quoted command), then I don't think $@ works